e2e-scripts/sippy-e2e-sippy-e2e-setup-commands.sh

```
@@ -266,23 +235,23 @@ spec:
-${KUBECTL_CMD} -n sippy-e2e describe job sippy-load-job
+echo "Waiting for sippy seed job to finish ..."
+${KUBECTL_CMD} -n sippy-e2e get job sippy-seed-job
+${KUBECTL_CMD} -n sippy-e2e describe job sippy-seed-job
 
 # We set +e to avoid the script aborting before we can retrieve logs.
 set +e
 
-echo "Waiting up to ${SIPPY_LOAD_TIMEOUT:=1200s} for the sippy-load-job to complete..."
-${KUBECTL_CMD} -n sippy-e2e wait --for=condition=complete job/sippy-load-job --timeout ${SIPPY_LOAD_TIMEOUT}
+echo "Waiting up to ${SIPPY_LOAD_TIMEOUT:=1200s} for the sippy-seed-job to complete..."
+${KUBECTL_CMD} -n sippy-e2e wait --for=condition=complete job/sippy-seed-job --timeout ${SIPPY_LOAD_TIMEOUT}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)